jbpm-db-scripts/src/test/java/org/jbpm/persistence/scripts/UpgradeScriptsTest.java

```diff
@@ -105,11 +108,39 @@ public void testExecutingScripts() throws IOException, SQLException {
         }
     }
 
+    /**
+     * Tests that DB schema is upgraded properly using database task_assigning_tables upgrade scripts.
+     * @throws IOException
+     * @throws SQLException
+     */
+    @Test
+    public void testTaskAssigningScripts() throws IOException, SQLException {
+        logger.info("entering testTaskAssigningScripts with type: {} ", distributionType);
+        try {
+            createSchema60UsingDDLs();
+            // We need to create optional table "PlanningTask" beforehand
+            ScriptFilter taskAssigningTable = filter("task_assigning_tables_")
+                                                .not("drop")
+                                                .setOptions(ScriptFilter.Option.DISALLOW_EMPTY_RESULTS, ScriptFilter.Option.THROW_ON_SCRIPT_ERROR);
+            executeScriptRunner(DB_DDL_SCRIPTS_RESOURCE_PATH, taskAssigningTable);
+            executeScriptRunner(DB_UPGRADE_SCRIPTS_RESOURCE_PATH, ScriptFilter.init(false, true).setDistribution(distributionType));
+            ScriptFilter scriptFilter = filter("task_assigning_tables.sql").
+                                        setDistribution(distributionType).
+                                        setOptions(ScriptFilter.Option.THROW_ON_SCRIPT_ERROR);
+            executeScriptRunner(DB_UPGRADE_SCRIPTS_RESOURCE_PATH, scriptFilter);
+            startAndPersistSomeProcess();
```

Comment:
Just one thought here: this test is just regression, nothing it's broken when upgrading the PlanningTask table, but there's no scenario using that table to check that it's working properly. wdyt?

Reply:
As discussed earlier, it's not really feasible to test here as it's part of integration repo. So leaving it as is